src/shared/string_op.c

```diff
@@ -759,6 +760,54 @@ long w_parse_time(const char * string) {
     return seconds >= 0 ? seconds : -1;
 }
 
+// Get time unit from seconds
+
+char*  w_seconds_to_time_unit(long seconds, bool long_format) {
+
+    if (seconds < 0) {
+        return "invalid";
```

Comment:
Shouldn't be useful to have all these strings defined in the header file? I think that probably could be used by the logs in some cases.

Reply:
Done. Every string and numeric value are now defined in **string_op.h** as a conversion utility